[09:53:24] Error: Telegram API Error  { code: 4, http_status_code: 502, message: 'Bad Gateway' }
[09:53:24] Error: Telegram API long poll problem. Code: 4
Bad Gateway
[09:53:28] Error: Telegram API Error  { code: 4, http_status_code: 502, message: 'Bad Gateway' }
[09:53:28] Error: Telegram API long poll problem. Code: 4
Bad Gateway
[13:28:36] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[13:28:36] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[14:00:37] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[14:00:37] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[14:32:39] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[14:32:39] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[15:04:28] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[15:04:28] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[15:36:06] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[15:36:06] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[16:07:41] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[16:07:41] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[16:39:04] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[16:39:04] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[17:10:26] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[17:10:26] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[17:41:42] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[17:41:42] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[18:12:55] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[18:12:55] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[18:15:47] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot/dist/vkbot.js:72:35)
[18:15:47] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[18:44:16] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[18:44:16] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[19:15:36] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[19:15:36] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[19:46:51] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[19:46:51] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[20:18:03] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[20:18:03] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[20:49:18] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[20:49:18] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[21:20:30] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[21:20:30] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[21:51:45] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[21:51:45] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[22:23:01] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[22:23:01] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[22:54:17] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[22:54:17] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[23:25:37] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[23:25:37] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[23:56:58] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[23:56:58] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
